packages/f-checkout/test/specs/component/f-checkout-delivery.component.spec.js

```diff
@@ -17,9 +17,22 @@ describe('f-checkout "delivery" component tests', () => {
         expect(CheckoutComponent.doesInputFieldExist(field)).toBe(true);
     });
 
-    forEach(['mobileNumber'])
-    .it('should display the mandatory fields', field => {
```

Comment:
does this mobileNumber test need to be separate from the fields test above it?

Reply:
Looking into this, I believe these are 2 separate tests. One is checking it is displayed and the other is if it exists in the dom. Doing a bit of digging now to see why it's only the mobileNumber is being tested.